### PR TITLE
finished editing header

### PR DESCRIPTION
**Pull request by pavlentiysv**
minor fixes of class names. added fonts

### CODE REVIEW

File: scss/components/_header.scss [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+    &:hover {
+      &:after {
+        content: "";
+        width: 100%;
+        height: 1px;
+        position: absolute;
+        bottom: 0px;
+        left: 0px;
+        background: linear-gradient(to right, #902616, #fe3c1f);
```
Comment (ElizavetaSelivanova) [inline scss/components/_header.scss]: Use variables for colors

File: scss/components/_header.scss [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    text-transform: uppercase;
+
+    &:hover {
+      &:after {
+        content: "";
+        width: 100%;
+        height: 1px;
+        position: absolute;
+        bottom: 0px;
+        left: 0px;
```
Comment (ElizavetaSelivanova) [inline scss/components/_header.scss]: No need write `px` here, just `bottom: 0, left: 0` =)

**Comment by angelinabylitskaya** [comment]
image
Logo is not centered

File: scss/components/_header.scss [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+  .header-container {
+    margin: 0 auto;
+    padding: 22px 0;
+    display: flex;
+    justify-content: center;
+  }
+
+  .logo {
+    font-size: 32px;
+    display: flex;
```
Comment (kobweb) [inline scss/components/_header.scss]: You can use mixin for flex containers

**Comment by angelinabylitskaya** [comment]
image

File: scss/components/_header.scss [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    align-items: center;
+    flex-direction: column;
+    margin: 0 34px;
+    img {
```
Comment (angelinabylitskaya) [inline scss/components/_header.scss]: Add an empty line before each selector

File: scss/components/_header.scss [hunk 5]
```diff
@@ -0,0 +1,63 @@
+header {
+  background: $color-header-bg;
+  color: $color-header-text;
+  font-family: "Raleway", sans-serif;
```
Comment (angelinabylitskaya) [inline scss/components/_header.scss]: Consider creating a variable for fonts.

File: scss/components/_slider.scss [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+    content: "";
+    display: inline-block;
+    width: 20px;
+    height: 20px;
+    border: solid $color-slider-arrow;
+    border-width: 2px 2px 0 0;
+  }
+}
+
+.slider__arrow_left {
```
Comment (angelinabylitskaya) [inline scss/components/_slider.scss]: You can user nesting here

File: scss/components/_slider.scss [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+        text-align: center;
+        text-decoration: none;
+        text-transform: uppercase;
+        color: $color-slider-text;
+        border: 1px solid $color-slider-text;
+        background-color: $color-slider-bg;
+        font-size: 16px;
+        font-weight: 200;
+        width: 150px;
+        height: 50px;
```
Comment (angelinabylitskaya) [inline scss/components/_slider.scss]: Do not set fixed height. Add paddings to button to align design height

File: scss/_fonts.scss [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: 'Myriad Pro';
+  src: url('../assets/fonts/MyriadPro-Regular.woff') format('woff');
+  font-weight: normal;
+  font-style: normal;
+}
```
Comment (angelinabylitskaya) [inline scss/_fonts.scss]: Leave a new line at the end of each file

**Pull request merged**